### PR TITLE
Add `isTampered` property

### PR DESCRIPTION
**Pull request by jvc9109**
## Summary
Add support for `isTampered` settings in KYCDocument. 
Add support for `isTampered` property in Indentity and Address matches.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

**Review by dara123 — DISMISSED** [review]

File: openapi/components/schemas/KycDocument/AddressMatches.yaml [hunk 1]
```diff
@@ -70,3 +70,7 @@ properties:
     nullable: true
     allOf:
       - $ref: ./KycDocumentSubtypes.yaml
+  isTampered:
+    description: Specifies if an address proof document is suspicious of being tampered.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycDocument/AddressMatches.yaml:74]: nitpick: 'suspicious' doesn't really work here

We should use this form if we are certain that the document has been tampered with 
```suggestion
    description: Specifies if an address proof document has been tampered with.
```
We should use this form if it's not certain that the document has been tampered with 
```suggestion
    description: Specifies if an address proof document may have been tampered with.
```

File: openapi/components/schemas/KycDocument/IdentityMatches.yaml [hunk 2]
```diff
@@ -101,6 +101,10 @@ properties:
     description: Specifies if an identity document has matching face proof.
     type: boolean
     example: false
+  isTampered:
+    description: Specifies if an identity document is suspicious of being tampered.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycDocument/IdentityMatches.yaml:105]: nitpick: 'suspicious' doesn't really work here

if we are certain that the document has been tampered with 
```suggestion
    description: Specifies if an identity document has been tampered with.
```
if we are not certain that the document has been tampered with 
```suggestion
    description: Specifies if an identity document may have been tampered with.
```

File: openapi/components/schemas/KycDocument/Settings/KycSettingsAddress.yaml [hunk 3]
```diff
@@ -56,6 +56,11 @@ properties:
         minimum: 0
         maximum: 100
         description: Weight added if the document is not one of the recognized document subtypes.
+      isTampered:
+        type: integer
+        minimum: 0
+        maximum: 100
+        description: Weight added if the document is suspicious of being tampered.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycDocument/Settings/KycSettingsAddress.yaml:63]: nitpick:
```suggestion
        description: Weight added if the document has been tampered with.
```

Comment (jvc9109) [reply openapi/components/schemas/KycDocument/Settings/KycSettingsAddress.yaml]: Thanks Chris! I have corrected the missing ones in the last commit.

File: openapi/components/schemas/KycDocument/IdentityMatches.yaml [hunk 4]
```diff
@@ -101,6 +101,10 @@ properties:
     description: Specifies if an identity document has matching face proof.
     type: boolean
     example: false
+  isTampered:
+    description: Specifies if an identity document may have been tampered with.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycDocument/IdentityMatches.yaml:105]: nitpick: consistency with the other description (last nitpick!)
```suggestion
    description: Specifies if an identity document has been tampered with.
```

**Review by noon-dawg — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**